psrm_utils.py

```diff
@@ -0,0 +1,20 @@
+import os
+import pickle
+from psrm_ci_ft_base import PSRM_CI_FT_BASE
+
+def cache_psrm(cache ='psrm.pkl', psrm_kwargs=None, force=False):
```

Comment:
Måske skal vi have den på den "traditionelle" form:
```suggestion
def cache_psrm(cache='psrm.pkl', force=False, *args, **kwargs):
```
og så sende `**kwargs` ind i `PSRM_CI_FT_BASE`.